### PR TITLE
feat: add home page

### PR DESCRIPTION
**Pull request by rahulbakshi4**
Feature Added:  
- Home Page

Components Added:
- Navbar
- Cards 
- Banner

### CODE REVIEW

File: src/components/Cards/cards.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    .product-img {
+      width: 15rem;
+      height: 12rem;
+    }
+    .home-cards {
+      gap: 2rem;
+    }
+    .h-card {
+      flex-direction: row;
+      padding: 2rem !important;
```
Comment (samyakshah3008) [inline src/components/Cards/cards.css:47]: It is not advisable to use !important 
You can fix it with any other alternative way

**Review by samyakshah3008 — APPROVED** [review]
Good folder structure and clean code
Nice naming conventions 

Suggestion: I have suggested minor suggestion as single comment, also you can include some comments in code where necessary 

Rest is good for approval ✅

**Comment by samyakshah3008** [comment]
You can also provide gif or live link in preview description for better code review:)

File: src/components/Cards/cards.css (hunk 1 above)
Comment (rahulbakshi4) [reply src/components/Cards/cards.css:47]: it was done to override properties

**Comment by rahulbakshi4** [comment]
> You can also provide gif or live link in preview description for better code review:)

Sure, will host it soon and will add preview link then

File: src/components/Cards/cards.css (hunk 1 above)
Comment (samyakshah3008) [reply src/components/Cards/cards.css:47]: oh no issues then ✅

File: src/App.css [hunk 2]
```diff
@@ -46,3 +50,18 @@ select {
 .bg-brown {
   background-color: rgb(var(--clr-brown)) !important;
```
Comment (Rohanmond) [inline src/App.css]: not sure about this but using import is not recommendable

**Review by Rohanmond — APPROVED** [review]
like your folder structure

File: src/App.css (hunk 2 above)
Comment (rahulbakshi4) [reply src/App.css]: Removed unwanted !important tag

**Pull request merged**